examples/todoapp/ios/Flutter/uatRelease.xcconfig

```diff
@@ -1,7 +1,6 @@
 #include? "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"
 #include "Generated.xcconfig"
 
-FLUTTER_TARGET=lib/main_uat.dart
```

Comment:
I can confirm that this change does not break any functionality.
This change is needed since when this target is specified the test does not start and the app idles.
Check following link under FAQ-> iOS -> `Running test stops on 'Wait for com.example.myapp to idle'`
https://leancodepl.docs.page/patrol/getting-started

Reply:
Do we need to adjust the rx_bloc_cli ?

Reply:
I believe that the app will function without this line and it should be safely removed. We are providing the entry point through the run configuration, so those lines won't be on any importance to the iOS project. We should remove this from the rx_bloc_cli because it is now redundant.